aptos-move/move-examples/staking/sources/oracle.move

```diff
@@ -0,0 +1,50 @@
+module staking::oracle {
+    use aptos_std::math128;
+    use pyth::i64;
+    use pyth::price;
+    use pyth::price_identifier;
+    use pyth::pyth;
+
+    friend staking::commission;
```

Comment:
Do we bother with `friend` or just go straight to using `package` dependencies?

Reply:
No longer needed the friendship